sql/core/benchmarks/JSONBenchmarks-results.txt

```diff
@@ -0,0 +1,33 @@
+================================================================================================
+Benchmark for performance of JSON parsing
+================================================================================================
+
+OpenJDK 64-Bit Server VM 1.8.0_163-b01 on Windows 7 6.1
+Intel64 Family 6 Model 94 Stepping 3, GenuineIntel
```

Comment:
With the same reason (https://github.com/apache/spark/pull/22845#discussion_r229199434), it's difficult to figure out the CPU.

Reply:
I'll make a PR to you.